<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Волновая спектроскопия</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Настройка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ax_input">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="value_ax">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Значение Аx</string>
   </property>
  </widget>
  <widget class="QLabel" name="value_ay">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Значение Аy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ay_input">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="df_input">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="value_df">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Значение df</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Значение dfp</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dfp_input">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="initial_angle_input">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="initial_angle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Начальный угол</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_angle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Конечный угол</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_angle_input">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>20</width>
     <height>921</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>130</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>1311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>771</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="set_values">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Измерение</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_video">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>540</y>
     <width>751</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="simulation">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>650</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Включить камеру</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_graph">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>650</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить график</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="set_progress_bar">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>780</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
